Phonebook.Frontend/src/app/pages/settings/settings.component.html

```diff
@@ -73,7 +57,7 @@ <h2 i18n="SettingsComponent|Layout Subtitle@@SettingsComponentSubTitleLayout">La
                 </mat-list-item>
             </mat-list>
         </div>
-        <div>
+        <div *ngIf=""> <!-- TODO: Check-if there are featureFlags -> add a function in .component.ts -->
```

Comment:
Why this have now an *ngIf?

Reply:
Here is it possible to write a function which checks how many feature Flags are in the code. *ngIf="true", run the code. 
- just preparation.

Reply:
@paule96 If you want @mschwrdtnr can write an issue to do this and remove it from the code

Reply:
Yes please do this @mschwrdtnr 
This is currently more confusing than helping...

Reply:
removed. #64